git-keeper-client/gkeepclient/client_configuration.py

```diff
@@ -15,22 +15,33 @@
 
 """Parses and provides access to the client configuration.
 
-The configuration file should be in the INI format:
+The configuration file must be stored at ~/.config/git-keeper/client.cfg
+
+The configuration file must be in the INI format:
 https://docs.python.org/3/library/configparser.html#supported-ini-file-structure
 
-There should be only one instance of the ClientConfiguration class. You can
-get it by calling the get_config() method provided by this module.
+This module stores a ClientConfiguration instance in the module-level variable
+named config. Call parse() on this instance before use. Attempting to call
+parse() a second time will raise an exception.
```

Comment:
So, the idea is that we call parse in main, and then anywhere else we just do the import?  Can this be clarified?